### ISSUE TITLE
[Feature Request] Bar chart axes ticks

### ISSUE THREAD

**Issue by Cobra5197** [neo4j-labs/neodash#172]
Hello,

Based on layout, I would love to be able to choose my own axis tick.

Cobra

**Comment by nielsdejong** [comment]
Hi @Cobra5197 ,
I'm finally getting around to picking this up. How do you imagine a tick axis customisation will look like?
Given a horizontal bar chart - For the x-axis, does this mean having a different string value returned to put as the tick string?

**Comment by Cobra5197** [comment]
Hi @nielsdejong ,
Yeah or a different scale. I usually use the bar chart to display the nodes count store so I would like to choose the scale on both axes to brighten up the visualization a bit.

**Comment by alfredorubin96** [comment]
@nielsdejong is this still relevant? Otherwise i'll close it

**Comment by Cobra5197** [comment]
Yes it's still relevant. For example when you have an integer, I want to display the axis every 'x'.
